osbuild/testutil/net.py

```diff
@@ -4,10 +4,13 @@
 """
 import contextlib
 import http.server
+import os
 import socket
 import ssl
+import sys
 import threading
```

Comment:
`osbuild/testutil/net.py:5:1: I001 [*] Import block is un-sorted or un-formatted`

What?